content/applications/finance/expenses/expense_reports.rst

```diff
@@ -0,0 +1,108 @@
+   only expenses with a status of :guilabel:`To Submit` that are *not* currently on another expense
+   report will appear in the newly created expense report.
+
+Once the expenses have been selected, click the :guilabel:`Create Report` button. The new report
+appears with all the expenses listed in the :guilabel:`Expense` tab. If there is a receipt attached
+to an individual expense, a :icon:`fa-paperclip` :guilabel:`(paperclip)` icon appears next to the
+:guilabel:`Customer to Reinvoice` and :guilabel:`Analytic Distribution` columns.
+
+When the report is created, the date range for the expenses appears in the :guilabel:`Expense Report
+Summary` field by default. It is recommended to edit this field with a short summary for each report
+to help keep expenses organized. Enter a short description for the expense report, such as `Client
+Trip NYC`, or `Office Supplies for Presentation`, in the :guilabel:`Expense Report Summary` field.
+
+Next, select a :guilabel:`Manager` from the drop-down menu to assign a manager to review the report.
+If needed, the :guilabel:`Journal` can be changed. Use the drop-down menu to select a different
+:guilabel:`Journal`.
+
+.. image:: expense_reports/expense-report-summary.png
+   :align: center
+   :alt: Enter a short description and select a manager for the report.
+
+If some expenses are missing from the report, they can still be added. Click :guilabel:`Add a line`
+at the bottom of the :guilabel:`Expense` tab. A pop up appears with all the available expenses that
+can be added to the report, with a status of :guilabel:`To Submit`. Tick the checkbox next to each
+expense being added, then click :guilabel:`Select`.
+
+The items now appear on the report that was just created. If a new expense needs to be added that
+does **not** appear on the list, click :guilabel:`New` to :ref:`create a new expense <expenses/new>`
+and add it to the report.
+
+.. image:: expense_reports/add-an-expense-line.png
+   :align: center
+   :alt: Add more expenses to the report before submitting.
+
+.. note::
+   Expense reports can be created in one of three places:
+
+   #. Navigate to the main :menuselection:`Expenses` app dashboard (also accessed by going to
+      :menuselection:`Expenses app --> My Expenses --> My Expenses`)
+   #. Navigate to :menuselection:`Expenses app --> My Expenses --> My Reports`
+   #. Navigate to :menuselection:`Expenses app --> Expense Reports`
+
+   In any of these views, click :guilabel:`New` to create a new expense report.
+
+.. _expenses/submit:
+
+Submit an expense report
+========================
+
+When an expense report is completed, the next step is to submit the report to a manager for
+approval. Reports must be individually submitted, and cannot be submitted in batches. Open the
+specific report from the list of expense reports, if the report is not already open. To view all
+expense reports, navigate to :menuselection:`Expenses app --> My Expenses --> My Reports`.
```

Comment:
This line (about opening all reports) should be above the "open the specific report"